src/components/registration/Registration.types.tsx

```diff
@@ -78,7 +83,7 @@ export type RegistrationLabel =
 export interface RegistrationStepContent<
   T extends RegistrationForms = RegistrationForms
 > {
-  subtitle?: string;
+  subtitle?: React.ReactNode;
```

Comment:
Tres mineur mais est ce que tu prefererais pas iporter React et n'écrire que "ReactNode" ?
```suggestion
  subtitle?: ReactNode;
```